concent_api/concent_api/urls.py

```diff
@@ -13,12 +13,28 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
+from django.conf      import settings
 from django.conf.urls import url, include
 from django.contrib   import admin
 
 import core.urls
+import gatekeeper.urls
 
-urlpatterns = [
-    url(r'^api/v1/', include(core.urls, namespace = 'core')),
-    url(r'^admin/',  admin.site.urls),
-]
+urlpatterns = []
+
+assert hasattr(settings, 'CONCENT_FEATURES') is True, "CONCENT_FEATURES variable must be defined in django settings."
+
+if "concent-api" in settings.CONCENT_FEATURES:
```

Comment:
Maybe we should add some kind of dictionary that would work as registry and auto load all urls from it ? What I am trying to point out is that at some point we might have a lot of those ifs. But maybe that can be added as separate feature, what do you think @cameel ?

Reply:
Sure. Sounds good. I'm fine with it being simple `if`s for now but having a dict that maps these names to URLs, required modules and other values is a good idea as well.

We could do something like this in some simple module that contains only definitions:

``` python
AVAILABLE_CONCENT_FEATURES = {
    "admin": {
        "required_django_apps": [
            "django.contrib.admin",
        ],
        "url_patterns":         [
            url(r'^admin/', admin.site.urls),
        ],
    },
    "concent-api": {
        "required_django_apps": [
            "core",
        ],
        "url_patterns":         [
            url(r'^api/v1/', include(core.urls, namespace = 'core')),
        ],
    },
    "gatekeeper": {
        "required_django_apps": [
            "gatekeeper",
        ],
        "url_patterns":         [
            url(r'^gatekeeper/', include(gatekeeper.urls, namespace = 'gatekeeper')),
        ],
    },
}
```